### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Jeday**
### Description

- [x] fixed missed export config
- [x] fixed build error
- [x] fixed missing helpers and exports  
- [x] Updated readme
- [x] Migration Guide 

### Checklist:

- [x]  Checked the changes locally.
- [ ]  Created/updated unit tests.
- [x]  Created/updated README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,21 +28,34 @@ const { LidoSDK } = require('@lidofinance/lido-ethereum-sdk');
...
+import { createPublicClient, http } from 'viem';
+import { goerli } from 'viem/chains';
+
+const client = createPublicClient({
+  chain: goerli,
+  transport: http(),
+});
 const sdk = new LidoSDK({
   chainId: 5,
   rpcProvider: ownRpcProvider,
```
Comment (DiRaiks) [inline README.md:52]: const is named `client`, but in props is using `ownRpcProvider`

File: packages/sdk/MIGRATION.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# Migrating from V2 -> V3
```
Comment (DiRaiks) [inline packages/sdk/MIGRATION.md:1]: It looks like you are describing a lot of internal changes that are not a migration guide, but simply describe what was done within version 3. I think the migration guide should only describe what the user needs to easily switch to version 3

File: packages/sdk/src/withdraw/claim/claim.ts [hunk 3]
```diff
@@ -1,9 +1,13 @@
 import { Logger, ErrorHandler } from '../../common/decorators/index.js';
-import { TransactionResult } from '../../core/types.js';
+import type {
+  NoCallback,
+  PopulatedTransaction,
+  TransactionResult,
+} from '../../core/types.js';
+import type { ClaimRequestsProps } from './types.js';
+import { encodeFunctionData, type SimulateContractReturnType } from 'viem';
```
Comment (solidovic) [inline packages/sdk/src/withdraw/claim/claim.ts:8]: (optional) order imports

File: packages/sdk/src/withdraw/claim/claim.ts [hunk 4]
```diff
@@ -29,5 +33,47 @@ export class LidoSDKWithdrawClaim extends BusModule {
     });
   }
 
-  // TODO Populate/Simulate
+  @Logger('Views:')
```
Comment (solidovic) [inline packages/sdk/src/withdraw/claim/claim.ts:36]: (optional) maybe change `@Logger('Views:')` ---> `@Logger('Views')` without two dots here and for each other loggers?

File: packages/sdk/src/withdraw/claim/claim.ts [hunk 5]
```diff
@@ -29,5 +33,47 @@ export class LidoSDKWithdrawClaim extends BusModule {
...
+  public async claimRequestsPopulateTx(
+    props: NoCallback<ClaimRequestsProps>,
+  ): Promise<PopulatedTransaction> {
+    const accountAddress = await this.bus.core.getWeb3Address(props.account);
+    const hints =
+      props.hints ??
+      (await this.bus.views.findCheckpointHints({
+        sortedIds: props.requestsIds,
+      }));
+    const { requestsIds } = props;
```
Comment (solidovic) [inline packages/sdk/src/withdraw/claim/claim.ts:66]: (my suggest to style guide): use `destructuring assignment` of `props` in top on function body

File: README.md [hunk 6]
```diff
@@ -28,21 +28,34 @@ const { LidoSDK } = require('@lidofinance/lido-ethereum-sdk');
 
 // Or, if you are using ES6+:
 import { LidoSDK } from '@lidofinance/lido-ethereum-sdk';
+
+// Or, import separate each module separately to save up on bundle size
+import { LidoSDKStake } from '@lidofinance/lido-ethereum-sdk/stake';
 ```
 
 ## Initialization
```
Comment (solidovic) [inline README.md:36]: Lets add `How to development/playground` segment to Readme. There is will be info how to prepare `SDK` and `playground`.

Something like:
```
yarn

yarn build

// Go to `playground/README.md` and make all from instruction 

yarn dev
```

File: packages/sdk/README.md [hunk 7]
```diff
@@ -84,74 +83,125 @@ The Lido Ethereum SDK consists of several modules:
...
+import { createPublicClient, http } from 'viem';
+import { goerli } from 'viem/chains';
+
+const client = createPublicClient({
+  chain: goerli,
+  transport: http(),
+});
 const sdk = new LidoSDK({
   chainId: 5,
   rpcProvider: ownRpcProvider,
```
Comment (DiRaiks) [inline packages/sdk/README.md:140]: https://github.com/lidofinance/lido-ethereum-sdk/pull/61/files#r1389037040

File: README.md (hunk 6 above)
Comment (Jeday) [reply README.md:36]: Added section.

File: packages/sdk/src/withdraw/claim/claim.ts (hunk 4 above)
Comment (Jeday) [reply packages/sdk/src/withdraw/claim/claim.ts:36]: Good Idea, but will touch this in next refactor concerning decorators and their usage.

File: README.md (hunk 1 above)
Comment (Jeday) [reply README.md:52]: fixed

File: packages/sdk/README.md (hunk 7 above)
Comment (Jeday) [reply packages/sdk/README.md]: fixed

**Review by solidovic — APPROVED** [review]

**Pull request merged**